packages/console/src/components/MfaFactorName/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { type AdminConsoleKey } from '@logto/phrases';
+import { MfaFactor } from '@logto/schemas';
+
+import DynamicT from '@/ds-components/DynamicT';
+
+const factorNameLabel: Record<MfaFactor, AdminConsoleKey> = {
+  [MfaFactor.TOTP]: 'mfa.totp',
+  [MfaFactor.WebAuthn]: 'mfa.webauthn',
+  [MfaFactor.BackupCode]: 'mfa.backup_code',
+};
+
+export type Props = {
+  type: MfaFactor;
+  agent?: string;
+};
+
+function MfaFactorName({ type, agent }: Props) {
+  return (
+    <>
+      <DynamicT forKey={factorNameLabel[type]} />
+      {agent && ` - ${agent}`}
```

Comment:
The "agent" is "User-Agent" which need to parse and format before display, would you mind creating an issue for this?

Reply:
Created.
LOG-7104